changelog.txt

```diff
@@ -15,6 +15,9 @@ that repo.
 
 ## New Scripts
 
+- `gui/workorder-details`: adjusts work orders' input item, material, traits
+- `assign-minecarts`: assign minecarts to hauling routes that don't have one
```

Comment:
the assign-minecarts line doesn't belong here. merge error?

Reply:
yeah, no idea how it got there.